.github/workflows/aws-deploy.yaml

```diff
@@ -12,70 +12,26 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
-      - uses: pre-commit/action@v2.0.0
-  sceptre-sandbox:
-    if: github.event_name == 'push'
+      - uses: pre-commit/action@v2.0.2
+
+  sceptre-deploy:
+    name: Deploy CloudFormation templates using sceptre
     runs-on: ubuntu-latest
     needs: pre-commit
+    if: github.ref == 'refs/heads/prod'
     steps:
-      - uses: actions/checkout@v2
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v1
-        with:
-          python-version: 3.8
-      - name: Cache python modules
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/pip
-          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
-          restore-keys: |
-            ${{ runner.os }}-pip-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install sceptre sceptre-ssm-resolver
+      - name: Checkout repository
+        uses: actions/checkout@v2
       - name: Assume AWS Role
         uses: aws-actions/configure-aws-credentials@v1
         with:
-          aws-access-key-id: ${{ secrets.CI_SERVICE_USER_ACCESS_KEY }}
-          aws-secret-access-key: ${{ secrets.CI_SERVICE_USER_SECRET_ACCESS_KEY }}
+          aws-access-key-id: ${{ secrets.NEXTFLOW_PROD_CI_ACCESS_KEY }}
```

Comment:
Maybe I'm behind on how this project has changed... Does it use nextflow?

Reply:
As far as I know, this project isn't related to Nextflow. These secret names were copied from what we did with for the [Nextflow accounts](https://github.com/Sage-Bionetworks-Workflows/aws-workflows-nextflow-infra/blob/prod/.github/workflows/aws-deploy.yaml). 

@thomasyu888: You can rename these to be more generic or specific to your project. You can either set these secrets at the org-level or the repo-level.

Reply:
Apologies @ahayden, @BrunoGrandePhD, I copy and pasted and forgot to change the values.